### PR TITLE
Fairly aggressive changes to the front page and logo.

### PR DESCRIPTION
**Pull request by mhredmond21**
You can see the changes here. http://abacus.freico.in:4000/

Please discuss this here. I'm not going to accept this unless we get a consensus.

### CODE REVIEW

**Comment by jtimon** [comment]
You've deleted the "no rents" feature. Maybe we should express it differently but I think that's the more important feature of them all.

**Comment by mhredmond21** [comment]
jtimon -- 

I'm trying to think about this functionally from the end user's POV and to simplify the landing page so there's less to click on and distract users. Google has done countless A/B studies how cluttered landing pages lead to lower click-through and downloads. We need to focus on what features does Freicoin have that Bitcoin does not. We need to be sure that these are immediately obvious to users.

I think these are

1) Permanent mining reward - an investment in "mining" capital only needs to worry about technology turning against them not the reward curve
2) Stable value - Freicoin will not attract as many speculators as bitcoin, so will enjoy a more stable (but lower) value
3) Long Term Thinking - because nobody is expecting the currency to appreciate, freicoin users will be more inclined to pick realistic investments than piratebonds. 

I did not remove your writings as an insult, only to make the landing have less information. I tried to incorporate things you and maaku wrote as much as I could in the about page. Feel free to make any changes you want to the about page on the main branch. I'm not sure if this branch will be merged or not yet. I really want to hear from maaku as well. Maybe it would be best if we can get on IRC at the same time but I know that is difficult.

I'd like to add a picture icon next to each feature to really drive it home. Even if that means removing more text.

Do you like the logo of the existing site, the logo I have on newlogo2 branch, or do you think we should have a different logo altogether?

Thanks, g

**Comment by maaku** [comment]
I agree that we should probably reduce the number of boxes on the front page to just three (and there's no reason the have the "Read more" link if we can fit all that info on the about page). However the "no rents" was the most important, albeit the most poorly worded so I can see why you took it off. I would say we should replace "Permanent mining reward" with a re-worked version of "No rents" that explains in layman's terms why a rent-free currency is a good thing.

For the logo, I prefer the one we have, but I defer input on this to my graphic designer, who is responsible for my contribution to the branding of Freicoin.

**Comment by jtimon** [comment]
I don't take this as an insult, I appreciate your efforts to make the site better.
I wasn't comfortable with removing the "read more" neither but I guess we can put that somewhere else. I'm sure that what I've written can be improved too.

Maybe we can join long term thinking with no rents. If we can't and we still want to show just 3 features, I would say perpetual reward is less important and I would prefer to remove that one.

I know it sounds strange the way is written. In my opinion I'm using the more accurate terms but I don't think many people are familiarized with the formal definition of economic rent. So you're right, we need to simplify it. But if we could only talk about one feature, I would talk about the removal of the capitalistic rents, which is what Gesell's works were mainly about. I see long term thinking just as a smart addition by Lietaer.
We need to think more about this, I'll visit the channel this afternoon.

EDIT: well, long term thinking also explain why capital-monies can be harmful not only to society but also to our relation with nature. I don't think we should merge them.

**Comment by maaku** [comment]
These changes have individually either been worked into the site, incorporated into pull-request #11, or mutually rejected; closing.
